views/templates/admin/tabs/content/listing.tpl

```diff
@@ -84,9 +83,13 @@
                                         <span class="switch_text switch-on" style="{if !$block['status']}display:none;{/if}">{l s='Activated' d='Modules.Blockreassurance.Admin'}</span>
                                         <span class="switch_text switch-off" style="{if $block['status']}display:none;{/if}">{l s='Deactivated' d='Modules.Blockreassurance.Admin'}</span>
                                     </label>
-
-                                    <span class="psre-edit" data-id="{$block['id_psreassurance']}"><i
-                                                class="material-icons">edit</i></span>
+                                  <div class="btn-group">
```

Comment:
look like there is something weird with the indentation level :thinking:

Reply:
:+1: